app/views/tasks/_form.html.erb

```diff
@@ -0,0 +1,20 @@
+<%= form_with model: @task, class: 'create-task' do |f|%>
+  <p>Please provide the following information to edit your task to our database:</p>
+  <div>
+    <%= f.label :name %>
+    <%= f.text_field :name %>
+  </div>
+
+  <div>
+    <%= f.label :description %>
+    <%= f.text_area :description %>
+  </div>
+
+  <div>
+    <%= f.label :completed_at %>
```

Comment:
Consider automatically populating `completed_at` when a task is created. Additionally, this does not need to be in the `edit` view since `mark_complete` takes care of changing this.